db/view/view.cc

```diff
@@ -678,7 +685,7 @@ struct value_getter {
 
 
 std::vector<view_updates::view_row_entry>
-view_updates::get_view_rows(const partition_key& base_key, const clustering_or_static_row& update, const std::optional<clustering_or_static_row>& existing) {
+view_updates::get_view_rows(const partition_key& base_key, const clustering_or_static_row& update, const std::optional<clustering_or_static_row>& existing, row_tombstone update_tomb) {
```

Comment:
I renamed it to `row_delete_tomb`, I hope it's more clear.
It's the tombstone from `update.tomb()` that we use to delete rows in `do_delete_old_entry`.
The problem and reason that I added a new parameter is that `do_delete_old_entry` calls `get_view_rows` with the `existing` row instead of `update`